src/routes/_components/status/StatusCard.html

```diff
@@ -0,0 +1,91 @@
+<a href="{url}" class="status-card" target="_blank" rel="noopener noreferrer">
+    <span class="card-title">
+     {title}
+    </span>
+  {#if hasBody}
+    <div class="card-content">
+  {#if imageUrl}
+    <LazyImage forceSize=true height="50" width="50" src={imageUrl} />
```

Comment:
Is there an `alt` we can pass in to this image?

Reply:
Alternatively, we can make it aria-hidden if it's just decorative. I don't know much about open graph.

Reply:
Yeah OK, I just checked a few, and it seems there is no alt. There is a `description`, but that seems more like the meta description tag. So we can just make it aria-hidden.